README.md

```diff
@@ -30,21 +30,10 @@ We also have a more detailed description of [features][features].
 
 Long instructions can be found [here][installation]. In short steps:
 
-1.  Make sure node-gyp is [installed correctly][node-gyp] with a python 2.7 version (and Visual Studio 2012 or 2013 if you are on Windows).
-
-2.  Install this package and [`mathjax-wrapper`][mathjax-wrapper]
-    for math rendering. Installation of mathjax-wrapper may take a long time.
-
-    ``` bash
-    apm install mathjax-wrapper
-    apm install markdown-preview-plus
-    ```
-
```

Comment:
@leipert  I removed this from the short instructions because if someone wants to look in the long instructions its there, but if you think it should be in README then please do add it back in :D